docs/appendices/environment_vars.rst

```diff
@@ -52,6 +52,8 @@ There are several environment variables that affect the way Toil runs.
 |                        | deleted until all associated nodes have been       |
 |                        | terminated.                                        |
 +------------------------+----------------------------------------------------+
+| TOIL_AZURE_ZONE        | The default region for provisioning instances.     |
```

Comment:
"Default" may be the wrong word here? It can change to a non-default region.

Reply:
"A specified region for provisioning instances."?